### PR TITLE
ChatGPT idea generator function

### PR DESCRIPTION
**Pull request by JustablockCode**
So i made Idea generator with chagpt... I made so it cant swear. This code is based on "idea generator" feature

### CODE REVIEW

**Comment by Ryan-shamu-YT** [comment]
We need this

**Comment by callumjt** [comment]
Dont use .innerHTML!!!

**Comment by callumjt** [comment]
And how do we know that api can be trusted?

**Comment by JustablockCode** [comment]
> And how do we know that api can be trusted?

yes... i use mubilop's API that is in popular gpt extension for turbowarp and penguinmod

**Comment by callumjt** [comment]
> > And how do we know that api can be trusted?
> 
> 
> 
> yes... i use mubilop's API that is in popular gpt extension for turbowarp and penguinmod

Just because its popular in another mod doesnt mean it can be trusted

And btw
> Dont use .innerHTML!!!

**Comment by JustablockCode** [comment]
> > > And how do we know that api can be trusted?
> > 
> > 
> > yes... i use mubilop's API that is in popular gpt extension for turbowarp and penguinmod
> 
> Just because its popular in another mod doesnt mean it can be trusted
> 
> And btw
> 
> > Dont use .innerHTML!!!

ok so...

File: features/features.json [hunk 1]
```diff
@@ -1207,5 +1207,17 @@
...
+  },
+    {
+    "title": "ChatGPT Ideas",
+    "description": "Get ideas for projects with ChatGPT on /ideas page!",
+    "credits": ["justablock", "MubiLop"],
+    "urls": [
+      "https://scratch.mit.edu/users/justablock",
+      "https://scratch.mit.edu/users/mubilop/"
+    ],
+    "file": "gpt-ideas",
```
Comment (MaterArc) [inline features/features.json:1219]: This should be changed to `gpt-ideas.js` and should be nested under scripts, in which it's also good to include a `runOn`...

Comment (JustablockCode) [reply features/features.json:1219]: but other projects have without .js and my code works fine(I tested it by downloading extension localy)

Comment (callumjt) [reply features/features.json:1219]: Ive just realised, this should be using v2, please read docs

File: features/gpt-ideas.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+.prepend(document.querySelector("div.banner-wrapper"));
+    document.querySelector("div.scratchtools.generator").onclick = function () {
+      generategpt();
+    };
+  }
+}
+
+function generategpt() {
+  const prompt = "make me 1 idea for scratch game... WITHOUT ANY CONTEXT! JUST IDEA AND NOTHING ELSE. AND DO NOT USE SWEAR WORDS OR BAD THEMES!";
+  const api_url = "https://reverse.mubi.tech/v1/chat/completions"
```
Comment (rgantzos) [inline features/gpt-ideas.js:21]: How trustworthy is this?

File: features/features.json (hunk 1 above)
Comment (rgantzos) [inline features/features.json:1219]: You should be using Feature v2, not the old version.

**Review by rgantzos — CHANGES_REQUESTED** [review]

**Pull request closed without merging**